portal/src/LinkToEtherScan.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useMemo } from "react";
+import ExternalLink from "./ExternalLink";
+import { parseEIP681 } from "./util/eip681";
+
+interface LinkToEtherScanProps {
+  className?: string;
+  addressInEIP681: string;
+}
+
+const LinkToEtherScan: React.VFC<LinkToEtherScanProps> = (props) => {
```

Comment:
Instead of a component, introduce a function to turn a EIP681 into a etherscan URL.

```typescript
function etherscan(eip681String: string): string
```

Put this function in `./util/eip681`